src/agent/libclusterfuzz/src/generated.rs

```diff
@@ -1,6 +1,5 @@
 // Rust generated by Microsoft, code originally from:
-//      https://github.com/google/clusterfuzz/blob/master/src/
-//          python/lib/clusterfuzz/stacktraces/constants.py
+//  https://github.com/google/clusterfuzz/blob/master/src/clusterfuzz/stacktraces/constants.py#:~:text=stack_frame_ignore_regexes
```

Comment:
How about making this a permalink?

Reply:
At the moment it would be a lie since the code is from some older version 🙂 

I need to double-check that we are up-to-date with latest.